omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -596,8 +596,18 @@
 default['private_chef']['bookshelf']['stream_download'] = true
 default['private_chef']['bookshelf']['access_key_id'] = "generated-by-default"
 default['private_chef']['bookshelf']['secret_access_key'] = "generated-by-default"
+default['private_chef']['bookshelf']['auth_check_disabled'] = false
 # Default: set to Host: header. Override to hardcode a url, "http://..."
 default['private_chef']['bookshelf']['external_url'] = :host_header
+default['private_chef']['bookshelf']['storage_type'] = :filesystem
+default['private_chef']['bookshelf']['db_pool_size'] = 20
+default['private_chef']['bookshelf']['db_pool_queue_max'] = 20
```

Comment:
I think we should consider increasing this substantially if we want to support a lot of concurrency out of the box.